README.md

```diff
@@ -60,28 +60,37 @@ The extension allows developers to find and fix security vulnerabilities in thei
 about the status of their code by continuously scanning it locally with [JFrog Xray](https://jfrog.com/xray/).
 
 ### What security capabilities do we provide?
-#### Software Composition Analysis (SCA)
-Scan your project dependencies for security issues.
-For selected security issues, get leverage-enhanced CVE data that is provided by our JFrog Security Research team.
-To learn more about enriched CVEs, see [here](https://www.jfrog.com/confluence/display/JFROG/JFrog+Security+CVE+Research+and+Enrichment)
+#### 📦 Software Composition Analysis (SCA)
+- Scan project dependencies for security issues 
+- Access enriched CVE data provided by the JFrog Security Research team.
 
-#### Advanced Scans
+Learn more about enriched CVEs in the [JFrog Security CVE Research and Enrichment](https://www.jfrog.com/confluence/display/JFROG/JFrog+Security+CVE+Research+and+Enrichment) documentation.
+
+#### 🏗️ Infrastructure as Code (IaC) Scanning 
+- Analyze Infrastructure as Code (IaC) files, such as Terraform, to identify security vulnerabilities and misconfigurations before deploying your cloud infrastructure.
+- Get actionable insights and recommendations for securing your IaC configurations.
+
+#### 🔐 Secrets Scanning
+Detect and prevent the inclusion of sensitive information, such as credentials and API keys, in your codebase.
```

Comment:
secrets and Iac are part of advance scans, it should be included where CA scan is.
In addition, maybe we should add images. we have similar informaiton on CA in the 'Viewing Vulnerability Details' section

Reply:
they also require the 
*Requires Xray version 3.66.5 or above and Enterprise X / Enterprise+ subscription with Advanced DevSecOps.*
since they are advance scans